### DISCUSSION TITLE
thanhle.blog/mot-so-thu-de-nhanh-vuot-qua-giai-doan-junior-fresher-hon

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#49] [Ideas]
# thanhle.blog/mot-so-thu-de-nhanh-vuot-qua-giai-doan-junior-fresher-hon

Một vài thứ mình thấy sẽ giúp ích để bạn vượt qua giai đoạn Junior/Fresher nhanh hơn

https://thanhle.blog/en/blog/mot-so-thu-de-nhanh-vuot-qua-giai-doan-junior-fresher-hon

**Comment by ngtrieuvi92** [comment]
viết hay đó em :D

**Comment by nguyenhoaikhang37** [comment]
Bài viết có ích quá anh ơi, yoloooo, đã upvote trên viblo của anh, mong anh sớm ra bài cách vượt qua fresher,junior nhanh

**Comment by binhnguyen2501** [comment]
😍

**Comment by thanhlmm** [comment]
Nice article about this topic:
- https://trangkienthuc.net/nguoc-chieu-thap-maslow-nguyen-tac-tu-ky-luat-ban-than-cuc-hieu-qua/
